tasks_week3_odelya_yaakowich/include/Animal.h

```diff
@@ -0,0 +1,75 @@
+/*********************
+Name: Odelya Yaakowich
+ID: 207346784
+**********************/
+
+
+#pragma once
+#include <iostream>
+#include <utility>
+#include "Location.h"
+
+#define left  0
+#define right  1
+#define up  2
+#define down  3
+
+class Animal {
+    std::string name;
+    Location location;
+    bool is_move;
+    int direction;
+
+public:
+
+    Animal(const std::string& name, const Location& location) : name(name), location(location), is_move(true) {} //ctor
+
```

Comment:
no use of virtual destructor 
check this
https://stackoverflow.com/questions/461203/when-to-use-virtual-destructors